spec/support/helpers/files_helper.rb

```diff
@@ -2,8 +2,11 @@ module Features
   module FilesHelper
     def attach_files(input, file_paths)
       file_input = page.find_field(input, type: :file, visible: :hidden)
-
-      file_input.attach_file(file_paths.map { |path| Rails.root.join(path) })
+      if file_paths.instance_of?(Array) && file_paths.length > 1
+        file_input.attach_file(file_paths.map { |path| Rails.root.join(path) })
+      else
+        file_input.attach_file(Rails.root.join(file_paths.first))
+      end
```

Comment:
Please, don't.
If you want to attach a single file, create an appropriate method, I can suggest `attach_file`.